mlflow/tracing/assessment.py

```diff
@@ -165,22 +165,22 @@ def log_feedback(
 
         This API is currently only available for `Databricks Managed MLflow <https://www.databricks.com/product/managed-mlflow>`_.
 
-    Logs a feedback to a Trace.
+    Logs feedback to a Trace.
 
     Args:
         trace_id: The ID of the trace.
         name: The name of the feedback assessment e.g., "faithfulness"
-        source: The source of the expectation assessment. Must be either an instance of
+        source: The source of the feedback assessment. Must be either an instance of
                 :py:class:`~mlflow.entities.AssessmentSource` or a string that
                 is a valid value in the
                 :py:class:`~mlflow.entities.AssessmentSourceType` enum.
-        value: The value of the expectation. It can be any JSON-serializable value.
+        value: The value of the feedback.
```

Comment:
The value for feedback *cannot* be any JSON-serializable value (e.g. nested lists / dicts aren't supported). However, I believe the type hint is sufficient here; I'd like to avoid copying the supported value information into the docstring, since it may easily become out of date.

I've filed a follow up ticket to change AssessmentValueType to FeedbackValueType, since expectations should indeed support a broader set of value types than feedback.